crates/loro-internal/src/kv_store.rs

```diff
@@ -1,8 +1,14 @@
 use bytes::Bytes;
 use std::{
+    collections::BTreeMap,
     ops::Bound,
     sync::{Arc, Mutex},
 };
+mod iter;
+mod mem_store;
+mod sstable;
+
+pub use mem_store::MemKvStore;
 
 pub trait KvStore: std::fmt::Debug + Send + Sync {
```

Comment:
This mod can be separated from the other things we're doing in loro-internal. Maybe we should extract them to its own crate?